README.md

```diff
@@ -26,30 +25,30 @@
 
```

Comment:
- [ ] Those links highlighted for removal or fixing previously are still there, broken. Kindly fix those links or remove them.

  ![image](https://github.com/BHObserver/To-Do-List/assets/54848363/2ca7a8a3-1ec1-47f8-aa69-0bfe6851396b)